### DISCUSSION TITLE
Hardening guide, API version refers to beta ?

### DISCUSSION THREAD

**Discussion by udf2457** [k3s-io/k3s#7051] [Q&A]
In the [hardening guide](https://docs.k3s.io/security/hardening-guide), the apiVersion refers to beta:

`apiVersion: pod-security.admission.config.k8s.io/v1beta1`

My (limited) understanding is that the this is incorrect and should really read:

`apiVersion: pod-security.admission.config.k8s.io/v1`

**Comment by brandond** [comment]
It depends on what version of k3s you're using. You should use the API version that is supported by the version of Kubernetes you're on.

**Comment by udf2457** [comment]
Fair enough @brandond, I wasn't aware of that (still a Kube noob), but the documentation makes a distinction between versions and config elsewhere so perhaps ought to continue that trend ?
